src/libraries/System.Net.Sockets/src/System/Net/Sockets/Socket.cs

```diff
@@ -2124,7 +2114,7 @@ public void EndConnect(IAsyncResult asyncResult)
         }
 
         public IAsyncResult BeginDisconnect(bool reuseSocket, AsyncCallback? callback, object? state) =>
-            TaskToApmBeginWithSyncExceptions(DisconnectAsync(reuseSocket).AsTask(), callback, state);
+            TaskToApm.Begin(DisconnectAsync(reuseSocket).AsTask(), callback, state);
```

Comment:
I don't understand how does this work without `TaskToApmBeginWithSyncExceptions` now.

Reply:
Ah ok, missed the discussion above.